DH_Engine/Classes/DHVoiceChatRoom.uc

```diff
@@ -11,7 +11,7 @@ var int SquadIndex;
 // NOTE: overridden to eliminate "has left channel" chat messages
 function RemoveMember(PlayerReplicationInfo PRI)
 {
-    if (PRI != none && PRI.VoiceID < 32 && IsMember(PRI, true))
+    if (PRI != none && PRI.VoiceID < 255 && IsMember(PRI, true))
```

Comment:
This `PRI.VoiceID` check doesn't really make sense in the context of its usage elsewhere. I would eliminate the check completely and just handle it in `IsMember`.